### PR TITLE
Remove pathdiv from stubbs converters

### PR DESCRIPTION
**Pull request by ShellyHerself**
Moses sent me these as files :roll_eyes:

### CODE REVIEW

File: tools_tag_converters/Convertor-Stubbs_Soso_to_Halo_OS_Soso.py [hunk 1]
```diff
@@ -62,7 +60,7 @@ def __init__(self, *args, **kwargs):
         self.resizable(0, 0)
 
         self.tags_dir = StringVar(self)
-        self.tags_dir.set(curr_dir + 'tags' + PATHDIV)
+        self.tags_dir.set(curr_dir + 'tags/')
```
Comment (csauve) [inline tools_tag_converters/Convertor-Stubbs_Soso_to_Halo_OS_Soso.py]: @MosesofEgypt Is this trailing `/` really necessary? Also shouldn't this be using `os.path.join(curr_dir, 'tags')`.

Comment (MosesofEgypt) [reply tools_tag_converters/Convertor-Stubbs_Soso_to_Halo_OS_Soso.py]: I'm pretty certain you can remove it from all the converters. I simply preserved the original code cause I was in a rush.

**Review by csauve — DISMISSED** [review]

**Review by MosesofEgypt — DISMISSED** [review]

File: tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py [hunk 2]
```diff
@@ -71,8 +69,17 @@ def convert_coll_tag(coll_path):
...
+            "flags",
+            "indirect_damage_material",
+            "body",
+            "shield",
+            "modifiers",
+            "pathfinding_box",
+            "pathfinding_spheres",
+            "nodes",
+            ):
+        tagdata[attr_name] = stubbs_tagdata[attr_name]
```
Comment (ShellyHerself) [inline tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py]: If you can type these out you might as well directly assign them.

File: tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py [hunk 3]
```diff
@@ -123,7 +130,7 @@ def __init__(self, *args, **kwargs):
         self.resizable(0, 0)
 
         self.tags_dir = StringVar(self)
-        self.tags_dir.set(curr_dir + 'tags' + PATHDIV)
+        self.tags_dir.set(curr_dir + 'tags/')
```
Comment (ShellyHerself) [inline tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py]: What's the point of always appending the '/'? Is it so you can avoid using path.join()?

What happens if our starting path has backslashes?

**Review by ShellyHerself — COMMENTED** [review]
There is an incredible amount of magic numbers in these files. And even though most of these were from before this PR. I am not comfortable merging it in this state.

This also hasn't been tested afaik. Not on linux at least. And I generally have low trust of files full of manual string operations on paths.

File: tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py (hunk 2 above)
Comment (MosesofEgypt) [reply tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py]: i did test the stubbs converters before i sent them to you. i had to use them to help that one user in Reclaimers. That's how I noticed they crashed on compile.

File: tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py (hunk 3 above)
Comment (MosesofEgypt) [reply tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py]: As I stated above to csauve, the slash can be removed. This is a remnant of old habits dying hard.

Comment (ShellyHerself) [reply tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py]: Could we handle that?

Comment (MosesofEgypt) [reply tools_tag_converters/Convertor-Stubbs_Coll_to_Halo_Coll.py]: done. also, just tested all converters on windows and opened converted tags in mozz. tags load and appear fine.

**Review by MosesofEgypt — DISMISSED** [review]

**Comment by ShellyHerself** [comment]
I'll be testing this later. I will be looking to get some test data as I don't have stubbz stuff on hand right now.

These changes are good so far

**Comment by ShellyHerself** [comment]
In the mean time, maybe something to look into. These converters have version number strings embedded in their window titles. Those obviously wouldn't be correct anymore now. Maybe define those at the top of the files in a constant like we have for the mek_installer right now.

**Review by csauve — DISMISSED** [review]

**Comment by ShellyHerself** [comment]
Testing these

**Review by csauve — APPROVED** [review]

**Review by ShellyHerself — COMMENTED** [review]
This worked on my end. I would like the file dialogs to be switched over to the ones in binilla some time. But for now this is good enough

**Comment by ShellyHerself** [comment]
Thank you for your effort, I appreciate that you changed so much

**Pull request merged**